R/metacoder.R

```diff
@@ -0,0 +1,158 @@
+#' Pivot sourmash_taxonomy_results to wide format
+#'
+#' @param taxonomy_annotate_tibble
```

Comment:
Is the `@` in comment lines designating that these are actions - such as a parameter, import, export, return?

Reply:
Here's some docs on how these roxygen function docs are structured: https://r-pkgs.org/man.html#roxygen-comments